test/integration/targets/user/vars/main.yml

```diff
@@ -10,4 +10,4 @@ status_command:
 
 default_user_group:
   openSUSE Leap: users
-  MacOSX: admin
+  MacOSX: staff
```

Comment:
The existing tests shouldn't need to be modified. Instead, add new tests to validate the bug you are fixing.

Reply:
As stated in issue #44316 teh default user group for MacOS is 'staff' instead of 'admin'

Reply:
Makes sense, did not notice this was fixing that issue.